veg2hab/vegkartering.py

```diff
@@ -259,6 +270,9 @@ def mozaiekregel_habtype_percentage_dict_to_string(
     if habtype_percentage_dict is None:
         return ""
 
+    # turn it back into a dict #TODO make this nicer
+    habtype_percentage_dict = {i[:3]: i[3] for i in habtype_percentage_dict}
```

Comment:
Bij het maken van de string wordt er over dict.items() geloopt dus dan kan daar ook gewoon de list in toch?